package.json

```diff
@@ -89,6 +89,7 @@
         "globals": "^15.15.0",
         "prettier": "^3.5.3",
         "rimraf": "^5.0.1",
+        "rollup": "^4.41.1",
```

Comment:
@neomorphic without this dependency the build on github kept failing with some @rollup error